packages/internals/src/sendPanic.ts

```diff
@@ -31,23 +43,11 @@ export async function sendPanic(error: RustPanic, cliVersion: string, engineVers
 
     const maskedSchema: string | undefined = schema ? maskSchema(schema) : undefined
 
-    let sqlDump: string | undefined
     let dbVersion: string | undefined
     // For a SQLite datasource like `url = "file:dev.db"` only schema will be defined
     const schemaOrUrl = schema || error.introspectionUrl
-    if (error.area === ErrorArea.INTROSPECTION_CLI && schemaOrUrl) {
-      let engine: undefined | IntrospectionEngine
-      try {
-        engine = new IntrospectionEngine()
-        sqlDump = await engine.getDatabaseDescription(schemaOrUrl)
```

Comment:
Are we sure that was not used for something and e.g. submitted with the error report?

Reply:
I had asked both Joel and Tom before removing `getDatabaseDescription` (which isn't implemented for MigrateEngine and isn't used anywhere else). The rationale is that we haven't made use of it in a long time (Tom never used it, Matthias did once or twice), and if we can remove legacy non-user-facing stuff, that's "always the best option"

Reply:
This referred to the resulting `sqlDump` - if we should not be using that to report to the error report. Might very well be that this system is actually set up to accept that and might benefit from doing so.